packages/main/src/popup-utils/PopoverRegistry.js

```diff
@@ -108,4 +127,19 @@ const getRegistry = () => {
 	return openedRegistry;
 };
 
+const getParentPopoversIfNested = instance => {
+	let currentElement = instance.parentNode;
+	const parentPopovers = [];
+
+	while (currentElement.parentNode) {
```

Comment:
I don't think we should do this. Traversing the DOM upwards and checking for a tag name is a prerequisite for many problems down the road. People can have their own popover-like components with different tag names, some might even not inherit from popover directly.

The point of the popup registry is to keep track of exactly this kind of information: whenever a new one opens, it registers itself, so we know which ones are below and which ones above. So to me the solution here is to completely use the registry.

Reply:
I agree with the fact that traversing the DOM is not a good thing.

In the case when someone makes popover-like component, with different tag and/or not extending ```ui5-popover``` the current registry is NOT going to care of it as well, so from this POV we are not going to brake anything we have supported.

The current goal is to make each ```ui5-popover``` aware whether it is opened inside of another one or not.